nss-ivory-trolls/BangazonSite
# added dropdown to products nav menu that links to views by product type

# Description

**Given** the user is on any page of the Bangazon site
**When** the user performs a gesture on a product category name
**Then** a list of all products in that category will be displayed, with title, quantity, and price/unit


## Type of change


# Testing Instructions for Change Made

```
git fetch -a
git checkout ab-ticket4
```
1. Confirm that "Products" nav item goes to the index list of all products when the word "Products" is clicked.
2. Confirm that each item in the dropdown next to Products nav item goes to the view of the specific product category and lists the products' titles, quantities, and prices.


# Checklist:
